### PR TITLE
Fix small typos on Read me

### PR DESCRIPTION
**Pull request by Riuqlav**
Fix small typos on Read me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
-GnuCash Web
-===========
+# GnuCash Web
```
Comment (joshuabach) [inline README.md:1]: Why use the hashtag notation? I like the underlines, as it is also readable in plaintext.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,18 @@
...
 
-*GnuCash Web* is a simple, easy to use, mobile-friendly webinterface for
+_GnuCash Web_ is a simple, easy to use, mobile-friendly webinterface for
```
Comment (joshuabach) [inline README.md:3]: _ and * do the same thing, and i think * is fine.

File: README.md [hunk 3]
```diff
@@ -26,37 +24,34 @@ Key features include:
 - Ease of use, especially on mobile
 
 | Browse account hierarchy                                                  | View and add transactions                                            |
-|---------------------------------------------------------------------------|----------------------------------------------------------------------|
+| ------------------------------------------------------------------------- | -------------------------------------------------------------------- |
```
Comment (joshuabach) [inline README.md:27]: Does this make a difference in the table formatting?

**Review by joshuabach — CHANGES_REQUESTED** [review]
Sorry for the late reply, I am a bit busy lately. Thanks for your contribution. 

Actually, only two of your suggested changes actually correct typos, the rest are stilistic changes most of which I see no reason for (except the removed spaces).

Can you explain the advantage of your suggestions?

**Comment by Riuqlav** [comment]
Oh yeah, my Prettier config corrects that way that on save, I actually just
went for the typos.
I don't really know why Prettier decides this is the better way.

**Pull request closed without merging**